python/manageJobs.py

```diff
@@ -291,6 +292,7 @@ def manageJobs(argv=None):
                 fprint("")
                 file_and_site_per_file = {}
                 file_and_site_per_file = find_input_file_site_per_job(
+                    blacklisted_sites = options.blacklistedSites + (parser_dict["manage"]["blacklistedsites"].split(",") if "blacklistedsites" in parser_dict["manage"].keys() else []),

```

Comment:
the `.prodconfig` value should go in the default value for the option (see other examples in this file)
also, `parser_dict` is a `defaultdict`, so checking for the key is not necessary

Reply:
Fixed